config.js

```diff
@@ -51,5 +51,10 @@ module.exports = {
       'shooting-clubs': 'https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=shootclub/',
       museums: 'https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=Museums/'
     }
+  },
+  pdf: {
+    url: env !== 'production' ?
+    `http://${process.env.PDF_CONVERTER_HOST}:${process.env.PDF_CONVERTER_PORT}/convert` :
+    process.env.PDF_CONVERTER_URL
```

Comment:
he's doing a bit of a dance for docker-compose there.

In prod we have another nginx with tls so no need for the port.

So you're right he should refactor the `docker-compse` template to do it there instead.